main.nf

```diff
@@ -133,16 +133,17 @@ process RunFastQC {
   publishDir "${directoryMap.fastQC}/${idRun}", mode: params.publishDirMode
 
   input:
-    set idPatient, status, idSample, idRun, file(fastqFile1), file(fastqFile2) from fastqFilesforFastQC
+    set idPatient, status, idSample, idRun, file(inputFile1), file(inputFile2) from inputFilesforFastQC
 
   output:
     file "*_fastqc.{zip,html}" into fastQCreport
 
   when: step == 'mapping' && !params.noReports
 
   script:
+  inputFiles = SarekUtils.hasExtension(inputFile1,"fastq.gz") ? "${inputFile1} ${inputFile2}" : "${inputFile1}"
```

Comment:
We should probably allow fq.gz as well right?

Reply:
We can think of it in the extractSample() function, but currently I can't think of anyone asking me about this particular `.fq.gz` extension.